tests/unit-tests/product/functions.php

```diff
@@ -740,6 +740,22 @@ public function test_wc_get_product() {
 		$this->assertEquals( $product->get_id(), $product_copy->get_id() );
 	}
 
+	/**
+	 * Test wc_get_product_classname().
+	 *
+	 * @since 3.9.0
+	 */
+	public function test_wc_get_product_classname() {
+		$this->assertInstanceOf( 'WC_Product_Simple', wc_get_product_classname( 'simple' ) );
+		$this->assertInstanceOf( 'WC_Product_Grouped', wc_get_product_classname( 'grouped' ) );
+		$this->assertInstanceOf( 'WC_Product_External', wc_get_product_classname( 'external' ) );
+		$this->assertInstanceOf( 'WC_Product_Variable', wc_get_product_classname( 'variable' ) );
+		$this->assertInstanceOf( 'WC_Product_Variation', wc_get_product_classname( 'variation' ) );
+
+		// Test incorrect type.
+		$this->assertInstanceOf( 'WC_Product_Simple', wc_get_product_classname( 'foo+bar' ), 10 );
```

Comment:
Why `10` is passed as the `$message` to `assertInstanceOf()`?

Reply:
No idea now, probably some test that I forget, thanks for review.